README.rst

```diff
@@ -10,11 +10,11 @@ This library is a very thin wrapper around `fissix
 to make Python 2 code more modern with the intention of eventually
 porting it over to Python 3.
 
-The ``python-modernize`` command works like `2to3
-<https://docs.python.org/3/library/2to3.html>`_. Here's how you'd rewrite a
+The ``python -m modernize`` command works like `fissix
+<https://github.com/jreese/fissix>`_. Here's how you'd rewrite a
```

Comment:
Maybe word this different, since fissix doesn't (currently) have a command line entrypoint like 2to3.

Reply:
```suggestion
The ``python -m modernize`` command works like
``python -m fissix``, see `fissix <https://github.com/jreese/fissix>`_.
Here's how you'd rewrite a
```

@jreese see also https://github.com/jreese/fissix/pull/24/files